src/www/campustreks/host.php

```diff
@@ -1,19 +1,43 @@
-<?php $publichtml = "c:/wamp64/www/campustreks/"; ?>
```

Comment:
path is specific to your working directory

Reply:
this exists across all of the .php webpage files. i believe we should merge this pull request, and then i can fix the $publichtml directory for each webpage in a separate pull request.